.github/actions/test-driver/action.yml

```diff
@@ -18,6 +18,9 @@ runs:
     - name: Build static viz frontend
       run: yarn build-static-viz
       shell: bash
+
+    - name: Setup upterm session
+      uses: lhotari/action-upterm@v1
```

Comment:
do we want to hang on to this? Or was it a one-time-use debugging thing?

(should probably rm the blank line (L21) if so)